### PR TITLE
Fix some warnings

### PR DESCRIPTION
**Pull request by remilauzier**

### CODE REVIEW

**Comment by kvark** [comment]
Thank you for the PR! Why are we deriving Debug everywhere?

**Comment by remilauzier** [comment]
I was just folowing the missing_debug lints but i can remove them if you prefer.

**Comment by kvark** [comment]
I don't find them particularly useful, tbh. It will affect the build times. If we need to debug-print anything, we'll add those derives.

File: src/render/object.rs [hunk 1]
```diff
@@ -103,6 +103,12 @@ impl InstanceDesc {
     }
 }
 
+impl Default for InstanceDesc {
```
Comment (kvark) [inline src/render/object.rs:106]: do we still need these?

**Review by kvark — COMMENTED** [review]
thank you!

File: src/render/object.rs (hunk 1 above)
Comment (remilauzier) [reply src/render/object.rs:106]: Dunno. Could always revert them and silence the warning if you prefer.

Comment (kvark) [reply src/render/object.rs:106]: Oh, I see now that this is about `Default`. I don't think these defaults do anything useful. If you don't mind, let's disable the warning on missing defaults - it's somewhat controversial.

**Pull request closed without merging**